patches/chrome-installer-util-BUILD.gn.patch

```diff
@@ -1,16 +1,22 @@
 diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
-index c66a6f6245ebe7d4abb1993c93f279e9e8442993..d78f0f1867d4b863133f826b45116dfe62813489 100644
+index c66a6f6245ebe7d4abb1993c93f279e9e8442993..eb0398139da2aa09472c9179de3320c96333bdff 100644
 --- a/chrome/installer/util/BUILD.gn
 +++ b/chrome/installer/util/BUILD.gn
-@@ -246,9 +246,16 @@ action("generate_strings") {
+@@ -246,9 +246,22 @@ action("generate_strings") {
      "$target_gen_dir/installer_util_strings.rc",
    ]
  
 +  brand = "$branding_path_component"
 +
-+  if (brave_chromium_build && !is_official_build) {
-+    # TODO(shong): Remove this.
-+    brand = brand + "-development"
++  if (brave_chromium_build) {
++    # When brave_strings.grd is modified, outputs should be re-generated.
```

Comment:
this script uses chromium_strings.grd as the grd file so why are we adding brave_strings.grd as a source?

Reply:
`//chrome/installer/util::generate_strings` uses 'create_string_rc.py`. and that script uses `brave_strings.grd`.

Reply:
That seems like a strange way to handle it. Can we mark this whole thing as a temporary workaround and not just the `-development` part? Open a new issue?

Reply:
`brave_strings.grd` is not related with brand. What do you mean?
Also, you think our changes in `create_string_rc.py` should be changed too?

Reply:
I'm saying that putting `brave_strings.grd` inside `create_string_rc.py` doesn't seem like the best way to handle this and that's what I'd like to open another ticket for.

Reply:
so we can merge this as-is, but I'd like to mark it as a workaround and open a ticket for the strings file issue

Reply:
I see. I'll create new issue for that.

Reply:
Added more comment and https://github.com/brave/brave-browser/issues/472 is created for this.

Reply:
Do you have more comment on this PR?

Reply:
sorry, no. I'll approve

Reply:
Thanks for kind review!